dlpy/tests/test_model.py

```diff
@@ -23,17 +23,21 @@
 #       the CASPROTOCOL environment variable.
 
 import os
+import shutil
 #import onnx
 import swat
 import swat.utils.testing as tm
 from swat.cas.table import CASTable
+from swat.cas.results import CASResults
 from dlpy.model import Model, Optimizer, AdamSolver, Sequence
+from dlpy.tensorboard import TensorBoard
```

Comment:
this would yield failures in tests if tensorboard is not installed, please take this into individual tests and skip if tensorboard is not installed